### PR TITLE
feat(publication): Add CuratingInstitutions to publication #NP-46344

### PR DESCRIPTION
**Pull request by torbjokv**

### CODE REVIEW

File: nva-datamodel-java/src/main/resources/publicationContext.json [hunk 1]
```diff
@@ -179,5 +179,10 @@
   "venues": {
     "@id": "venue",
     "@container": "@set"
+  },
+  "curatingInstitutions": {
+    "@id": "curatingInstitution",
```
Comment (torbjokv) [inline nva-datamodel-java/src/main/resources/publicationContext.json:184]: @brinxmat "@id" should maybe be organization?

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java [hunk 2]
```diff
@@ -390,6 +394,14 @@ public boolean satisfiesFindableDoiRequirements() {
         return FindableDoiRequirementsValidator.meetsFindableDoiRequirements(this);
     }
 
+    public Set<URI> getCuratingInstitutions() {
+        return nonNull(curatingInstitutions) ? curatingInstitutions : Collections.emptySet();
```
Comment (sondrev) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java:398]: Why not just return null? There is a difference "not set" and "no curating institutions"

**Review by sondrev — CHANGES_REQUESTED** [review]
1. see coment
2. bump version number for datamodel
3. You need to update PublicationResponse aswell

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java (hunk 2 above)
Comment (brinxmat) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java:398]: @sondrev For the sake of the receiver, in serialization

**Review by brinxmat — DISMISSED** [review]

**Pull request merged**